docker/package/packages.py

```diff
@@ -543,6 +546,6 @@ def mk_rollup_package(name, proto):
     ]
 
 
-packages.extend(mk_rollup_packages("sc", ["PtMumbai"], "smart"))
+packages.extend(mk_rollup_packages("sc", ["PtMumbai", "PtNairob"], "smart"))
```

Comment:
It looks like `tx` rollup binaries are gone for good now, so we should update this to reflect that.

OTOH the current code works and this isn't urgent, so we can do this change later as well